src/client/src/sync/controller.rs

```diff
@@ -111,21 +109,20 @@ where
                             match update_result {
                                 Ok(update) => {
                                     if let Err(err) = self.process_updates(update).await {
-                                        error!("processing updates: {}", err);
+                                        error!("Error processing updates: {}", err);
```

Comment:
this is already error, so marking as error seems to be redundant.  However, welcome making error clear...